src/ar/arabanime/src/eu/kanade/tachiyomi/animeextension/ar/arabanime/ArabAnime.kt

```diff
@@ -0,0 +1,264 @@
+package eu.kanade.tachiyomi.animeextension.ar.arabanime
+
+import android.app.Application
+import android.content.SharedPreferences
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.preference.ListPreference
+import androidx.preference.PreferenceScreen
+import eu.kanade.tachiyomi.animeextension.ar.arabanime.dto.AnimeItem
+import eu.kanade.tachiyomi.animeextension.ar.arabanime.dto.Episode
+import eu.kanade.tachiyomi.animeextension.ar.arabanime.dto.PopularAnimeResponse
+import eu.kanade.tachiyomi.animeextension.ar.arabanime.dto.ShowItem
+import eu.kanade.tachiyomi.animesource.ConfigurableAnimeSource
+import eu.kanade.tachiyomi.animesource.model.AnimeFilter
+import eu.kanade.tachiyomi.animesource.model.AnimeFilterList
+import eu.kanade.tachiyomi.animesource.model.AnimesPage
+import eu.kanade.tachiyomi.animesource.model.SAnime
+import eu.kanade.tachiyomi.animesource.model.SEpisode
+import eu.kanade.tachiyomi.animesource.model.Video
+import eu.kanade.tachiyomi.animesource.online.AnimeHttpSource
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.util.asJsoup
+import kotlinx.serialization.json.Json
+import okhttp3.FormBody
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Request
+import okhttp3.Response
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+import java.util.Base64
+
+class ArabAnime: ConfigurableAnimeSource, AnimeHttpSource() {
+
+    override val name = "ArabAnime"
+
+    override val baseUrl = "https://www.arabanime.net"
+
+    override val lang = "ar"
+
+    override val supportsLatest = true
+
+    private val json = Json {
+        ignoreUnknownKeys = true
+    }
+
+    private val preferences: SharedPreferences by lazy {
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
+    }
+
+    // ============================== Popular ===============================
+    @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
We support Android 6 so this is an issue

Reply:
i hope this fix it